peerscout_api/recomend_editor.py

```diff
@@ -0,0 +1,129 @@
+import os
```

Comment:
It might be good to have another lower priority task to refactor the notebook(s) to use the functions that you have extracted. In that case, it would maybe make sense for this module to live in `data_science_pipeline.peerscout`. Although it could be moved at that time.